reference/test/preconditioner/isai_kernels.cpp

```diff
@@ -564,15 +844,14 @@ TYPED_TEST(Isai, UseWithIluPreconditioner)
     using UpperIsai = typename TestFixture::UpperIsai;
     const auto vec = gko::initialize<Dense>({128, -64, 32}, this->exec);
     auto result = Dense::create(this->exec, vec->get_size());
-    auto mtx =
-        gko::share(Dense::create_with_config_of(gko::lend(this->l_dense)));
-    this->l_dense->apply(gko::lend(this->u_dense), gko::lend(mtx));
+    auto mtx = gko::share(Dense::create_with_config_of(lend(this->l_dense)));
+    this->l_dense->apply(lend(this->u_dense), lend(mtx));
     auto ilu_factory = gko::preconditioner::Ilu<LowerIsai, UpperIsai, false,
                                                 index_type>::build()
                            .on(this->exec);
     auto ilu = ilu_factory->generate(mtx);
 
-    ilu->apply(gko::lend(vec), gko::lend(result));
+    ilu->apply(lend(vec), lend(result));
```

Comment:
I remember we decide to use `gko::lend` not `lend` if there is no `using gko::lend` or `using namespace gko`

Reply:
Are you arguing from a technical or a code-style viewpoint? `lend` is still found during Argument-Dependent Lookup (ADL), since the template parameter of `std::unique_ptr<gko::Dense>` is in the `gko` namespace, so its contents get added to the overload resolution candidate set.

Reply:
I am with @yhmtsai here. I know that ADL used otherwise, but I would rather have the explicit `gko::` in front, so it is clear where it is comming from.

Reply:
We are currently not really consistent in that regard, `[^:]lend` and `gko::lend` are both used throughout our tests. I personally prefer `lend` over `gko::lend` and `.get()` because it is a) as shorter without namespace and b) introduces our ownership semantics better than .get()